### PR TITLE
Cart/pipelines page

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Re-integrate testing for launching pipelines.

## Related issue
#192 

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

~~* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~~
~~* [ ] Tests added (or description of how to test) for any new features.~~
~~* [ ] User documentation updated for UI or technical changes.~~

### CODE REVIEW

File: src/main/java/ca/corefacility/bioinformatics/irida/ria/web/pipelines/PipelineController.java [hunk 1]
```diff
@@ -136,50 +136,6 @@ public PipelineController(SequencingObjectService sequencingObjectService,
 		this.iridaPipelinePluginStyle = iridaPipelinePluginStyle;
```
Comment (apetkau) [inline src/main/java/ca/corefacility/bioinformatics/irida/ria/web/pipelines/PipelineController.java]: It looks like `iridaPipelinePluginStyle` is no longer used in this class. Could it be removed?

Comment (joshsadam) [reply src/main/java/ca/corefacility/bioinformatics/irida/ria/web/pipelines/PipelineController.java]: Removed in d210b2c8a786a046301efaa93f37a2bbc0e3229a

Comment (apetkau) [reply src/main/java/ca/corefacility/bioinformatics/irida/ria/web/pipelines/PipelineController.java]: Thanks @joshsadam.

I just noticed there's two methods which are unused in this class too `extractPipelineParameters` and `getCartSummaryMap`. Could you check to make sure these aren't required and remove them if so?

Comment (joshsadam) [reply src/main/java/ca/corefacility/bioinformatics/irida/ria/web/pipelines/PipelineController.java]: Good catch, removed in aa2c631d9e276eb6e2b17542e8ce42304eac1a82

Comment (apetkau) [reply src/main/java/ca/corefacility/bioinformatics/irida/ria/web/pipelines/PipelineController.java]: Looks great. Thanks @joshsadam

**Pull request merged**